plugins/org.eclipse.n4js.json.model/src/org/eclipse/n4js/json/model/utils/JSONModelUtils.java

```diff
@@ -52,17 +53,20 @@
 	/** The JSON file extension. */
 	public static final String FILE_EXTENSION = "json";
 
+	/**
+	 * If the given JSON value is a {@link JSONBooleanLiteral} with a value of <code>true</code>, returns
+	 * <code>true</code>, otherwise <code>false</code>.
+	 */
+	public static boolean asBoolean(JSONValue jsonValue) {
```

Comment:
`asBooleanOrFalse` to be consistent with `asStringOrNull` below?

Reply:
done